### PR TITLE
AI requires silicon exp instead of crew exp for roundstart role

### PR DESCRIPTION
**Pull request by ohnopigeons**
:cl:
tweak: AI now requires silicon playtime instead of crew playtime for roundstart role
/:cl:

I tire of AIs that ask for their laws to be purged, don't know how to robot chat, don't know how to unlock borgs, and don't know how to make announcements, etc. Sometimes they are all of these things at once.

For job exp purposes (and only for these purposes), AI counts as a head role.

### CODE REVIEW

**Comment by Kerbin-Fiber** [comment]
Now we just have to hope Revs don't go and kill the AI thinking it's a head like they do with the QM.

**Comment by Anonmare** [comment]
DEATH TO THE AI BOURGEOIS

**Comment by ShizCalev** [comment]
conflict.

File: code/modules/jobs/job_types/silicon.dm [hunk 1]
```diff
@@ -11,9 +11,10 @@ AI
 	selection_color = "#ccffcc"
 	supervisors = "your laws"
 	req_admin_notify = 1
-	minimal_player_age = 30
+	minimal_player_age = 14
```
Comment (lordpidey) [inline code/modules/jobs/job_types/silicon.dm]: I'm mildly against this being lowered.

Comment (pigeonsk) [reply code/modules/jobs/job_types/silicon.dm]: All other head restrictions have age limit set to 14. Even the HoS which has, from what I can tell, the most restrictive requirements has it set to 14. 30 days for the AI seems anomalous and I can only surmise it as a workaround for not requiring silicon exp.

Comment (optimumtact) [reply code/modules/jobs/job_types/silicon.dm]: no, it's because it's probably the single hardest role you can play on the station

Comment (MrStonedOne) [reply code/modules/jobs/job_types/silicon.dm]: Keep the the age, the ai is the most grief capable role and the hardest to take down ICly when admins aren't on, and the point of its long time is to give value to having an account with good standing.

Our servers base it off of byond account age and not server account age anyways, so its not that big of a deal to lock out byond accounts under a month old for a grief capable role like AI.

**Comment by KorPhaeron** [comment]
Since MSO hasn't replied I'm going to assume he doesn't want this, he is welcome to bug me to reopen if I'm mistaken

**Comment by lordpidey** [comment]
MSO has posted on this since you added him.  It seemed he had issue with the age being 14 rather than 30.

The age has been changed back to 30.

**Comment by ExcessiveUseOfCobblestone** [comment]
So for all intensive purposes code-wise (with the exception of rev) the AI is now a command job?

**Comment by pigeonsk** [comment]
Not all
>For job exp purposes (and only for these purposes)

**Pull request merged**